raiden/api/rest.py

```diff
@@ -159,6 +166,8 @@ def __init__(self, raiden_api):
         self.tokens_list_schema = TokensListSchema()
         self.partner_per_token_list_schema = PartnersPerTokenListSchema()
         self.transfer_schema = TransferSchema()
+        self.connection_connect_schema = ConnectionConnectSchema()
+        self.connection_leave_schema = ConnectionsLeaveSchema()
```

Comment:
Actually I just realized that I am not using these schemes anywhere anymore, so I might as well just remove this.